lib/spack/spack/test/cmd/ci.py

```diff
@@ -155,7 +155,7 @@ def test_ci_generate_with_env(
 ):
     """Make sure we can get a .gitlab-ci.yml from an environment file
     which has the gitlab-ci, cdash, and mirrors sections."""
-    mirror_url = "https://my.fake.mirror"
+    mirror_url = "file://my.fake.mirror"
```

Comment:
This is not a valid file URL, requires an absolute path not a relative path `my.fake.mirror`.

Reply:
Let me clean up this test file a bit while I am at it.